httpie/input.py

```
@@ -0,0 +1,693 @@
+
+        target.append((item.key, value))
+
+    return RequestItems(headers_class(headers),
+                        data_class(data),
+                        files_class(files),
+                        params_class(params))
+
+
+def readable_file_arg(filename):
+    try:
+        open(filename, 'rb')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)